formats/service_manual_service_standard/publisher/details.json

```diff
@@ -3,29 +3,9 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
-    "points": {
-      "type": "array",
-      "items": {
-        "type": "object",
-        "properties": {
-          "title": {
-            "type": "string",
-            "description": "The title of the point. It matches the title of the relevant page."
-          },
-          "summary": {
-            "type": "string",
-            "description": "The summary of the point. It matches the summary on the relevant page."
-          },
-          "base_path": {
-            "$ref" : "#/definitions/absolute_path"
-          }
-        },
-        "required": [
-          "title",
-          "summary",
-          "base_path"
-        ]
-      }
+    "body": {
+      "type": "string",
+      "description": "A single line of text that serves as a second, less prominent introduction."
```

Comment:
It looks like we added the test here: https://github.com/alphagov/service-manual-frontend/commit/9e679c59

The test was just missing in master.